src/components/__tests__/__snapshots__/Header.spec.js.snap

```diff
@@ -2,16 +2,1454 @@
 
 exports[`Header renders correctly 1`] = `
 Array [
+<<<<<<< HEAD
+  .emotion-65 {
+=======
   .emotion-33 {
+>>>>>>> b61bcd7f034ce46a961d06edd8828a5c1905eab7
```

Comment:
I notice a lot of these snapshots have these conflict indicators in here. I'm wondering if this is part of the problem. After adding the `react-dom` mock, could you run `npm test -- -u` to re-update these and see if tests pass for you?

Reply:
Nailed it! 

Not sure how those merge conflict came up. Seemed odd to have them in snapshots of all places. I had run the alias `test:update`, but maybe the mock file wasn't properly in place at the time and I forgot to run another snapshot update _after_ I'd put the mock in. 

Clearly missed loads when trying to resolve in e2daec8, but that kinda pointed us in the right direction in the end.

Sorted now! 🎉  Thanks a lot for stepping through this with me.

Reply:
🎉 🎉 🎉 🎉 🎉 

So glad that worked! Glad I could help!